packages/block-library/src/image/image.js

```diff
@@ -555,6 +557,14 @@ export default function Image( {
 										lightbox: { enabled: newValue },
 									} );
 								} }
+								disabled={ lightboxToggleDisabled }
+								help={
```

Comment:
I'm not sure if this is the best way to add the help text — any thoughts on this appreciated!

Reply:
As far as I can tell, this is indeed the appropriate way 👍